src/Web/ViewModels/Account/LoginViewModel.cs

```diff
@@ -17,5 +19,13 @@ public class LoginViewModel
 
         [Display(Name = "Remember me?")]
         public bool RememberMe { get; set; }
+
```

Comment:
You'll see in the latest code (after you would have pulled latest from master maybe yesterday?) that you just now need the 

```public DeviceFingerPrintingViewModel DeviceFingerPrinting { get; set; }```

It has the client* props on it.